alcs-api/src/application/application.controller.ts

```diff
@@ -45,14 +44,8 @@ export class ApplicationController {
 
   @Get()
   @UserRoles(...ANY_AUTH_ROLE)
-  async getAll(@Query('dm') dm?: string): Promise<ApplicationDto[]> {
-    let decisionMaker;
-    if (dm) {
-      decisionMaker = await this.codeService.fetchDecisionMaker(dm);
-    }
-    const applications = await this.applicationService.getAll({
-      decisionMaker,
-    });
+  async getAll(): Promise<ApplicationDto[]> {
+    const applications = await this.applicationService.getAll({});
```

Comment:
is it possible to mark filter as optional? So if you need to get all values it will looks like this: ....applicationService.getAll();

Reply:
Absolutely, not sure why I didn't do that in the first place lol.

Reply:
Done!